ci_scripts/sync_board_database.py

```diff
@@ -25,12 +25,12 @@
 from mbed_tools_lib.exceptions import ToolsError
 from mbed_tools_lib.logging import log_exception, set_log_level
 
-from mbed_targets._internal.target_database import SNAPSHOT_FILENAME
-from mbed_targets.mbed_targets import MbedTargets
+from mbed_targets._internal.board_database import SNAPSHOT_FILENAME
+from mbed_targets.boards import Boards
 
 logger = logging.getLogger()
 
-TARGET_DATABASE_PATH = Path(
+BOARD_DATABASE_PATH = Path(
```

Comment:
Should the Path argument be 'mbed_boards' instead of 'mbed_targets'?

Reply:
Ah no, here it refers to 'mbed_targets' the top level directory name rather than `mbed_targets` a variable name. This hasn't changed since the package is still `mbed-targets`.